TrafficCapture/dockerSolution/build.gradle

```diff
@@ -117,3 +117,9 @@ task buildDockerImages {
 }
 
 tasks.getByName('composeUp').dependsOn(tasks.getByName('buildDockerImages'))
+
+java {
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(11) // Locks supported Java version to 11
```

Comment:
This probably won't do much since the dockerSolution project doesn't build Java code. This would need to be added for each project. Is there a way to do this at a global level, for all the Gradle projects with something like Gradle settings?

Reply:
You're right, this was failing when I run `./gradlew dockerSolution:build` if I had a different version specified, but I guess it's a habit of mine that I run build before running `./gradlew dockerSolution:composeUp` or down here.
Thanks Greg, I'll address this.